### PR TITLE
Update to latest spacy and automate evaluation

### PR DESCRIPTION
**Pull request by eric-01001000**

### CODE REVIEW

**Review by kurellajunior — COMMENTED** [review]
i don’t see how this runs two different spacy versions

File: evaluate.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    # 2. text-based
+    HIT_RATE=$(python -m tests.evaluate_sentence_splitter "$DATA_DIR" --spacy-model "$MODEL_NAME" --no-split-on-line-breaks)
+    printf '%s\ttext-based\t%s\t%s\t\n' "$SPACY_VERSION" "$MODEL_SIZE" "$HIT_RATE"
+  done
+
+  # don't update spacy model again if we're already on the latest version
+  [ $i = "latest" ] && break
+
+  # update spacy model
+  poetry add spacy@latest --quiet
```
Comment (kurellajunior) [inline evaluate.sh:44]: oi, that is tricky, now I got it

File: evaluate.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+done
```
Comment (kurellajunior) [inline evaluate.sh:45]: could just add a git reset --fileame here to undo back to the current spacy version

Comment (eric-01001000) [reply evaluate.sh:45]: yes, followed by poetry install. i'll add it.

Comment (kurellajunior) [reply evaluate.sh:46]: much better :heart:

**Review by kurellajunior — APPROVED** [review]

File: evaluate.sh (hunk 1 above)
Comment (kurellajunior) [reply evaluate.sh]: meant to be here: much better :heart:

**Pull request merged**